Formula/e/emqx.rb

```diff
@@ -38,20 +38,36 @@ class Emqx < Formula
 
   def install
     ENV["PKG_VSN"] = version.to_s
+    ENV["BUILD_WITHOUT_QUIC"] = "1"
     touch(".prepare")
-    system "make", "emqx"
+    system "make", "emqx-rel"
     prefix.install Dir["_build/emqx/rel/emqx/*"]
     %w[emqx.cmd emqx_ctl.cmd no_dot_erlang.boot].each do |f|
       rm bin/f
     end
     chmod "+x", prefix/"releases/#{version}/no_dot_erlang.boot"
     bin.install_symlink prefix/"releases/#{version}/no_dot_erlang.boot"
+    return unless OS.mac?
+
+    # ensure load path for libcrypto is correct
+    crypto_vsn = Utils.safe_popen_read("erl", "-noshell", "-eval",
+                                       'io:format("~s", [crypto:version()]), halt().').strip
+    libcrypto = Formula["openssl@3"].opt_lib/shared_library("libcrypto", "3")
+    %w[crypto.so otp_test_engine.so].each do |f|
+      dynlib = lib/"crypto-#{crypto_vsn}/priv/lib"/f
+      old_libcrypto = dynlib.dynamically_linked_libraries
```

Comment:
Sorry, this should fix the build error:
```suggestion
      old_libcrypto = dynlib.dynamically_linked_libraries(resolve_variable_references: false)
```